Snakefile

```diff
@@ -2,21 +2,30 @@ import glob
 import os
 import sys
 from snakemake.remote.HTTP import RemoteProvider as HTTPRemoteProvider
-HTTP = HTTPRemoteProvider()
 
 sys.path.insert(0, workflow.basedir)
 from constants import VERSION
 
-print(f"Ploidetect-pipeline {VERSION}")
+HTTP = HTTPRemoteProvider()
+MEM_PER_CPU = 7900
 
+print(f"Ploidetect-pipeline {VERSION}")
 
-## Load config values
-configfile: os.path.join(workflow.basedir, "CONFIG.txt")
 
+## Load default config values
+# - loading type errors like 'list indices must be integers or slices, not str'
+#   probably means the configfile given has a list where defaults were a dict.
+#   eg. config['chromosomes'] as list vs config['chromosomes']['hg38'] as list
+configfile: os.path.join(workflow.basedir, "resources/config/default_run_params.yaml")
+configfile: os.path.join(workflow.basedir, "resources/config/default_case.yaml")
+configfile: os.path.join(workflow.basedir, "resources/config/genome_ref.yaml")
```

Comment:
@lculibrk - these 3 new files are base off CONFIG.txt
Separating out the patient bams and parameters from other details.